core/src/worker/workflow/managed_run/managed_wf_test.rs

```diff
@@ -0,0 +1,194 @@
+use super::*;
```

Comment:
This file is named `..._test.rs` but does not appear to contain any tests.

Reply:
The code is only in support of tests. The whole module is behind a `cfg(test)`